Sources/AppLovinSDKResources/AppLovinSDKResources.swift

```diff
@@ -0,0 +1,9 @@
+//
+//  AppLovinSDKResources.swift
+//  AppLovinSDK
+//
+//  Created by Ritam Sarmah on 8/2/24.
+//  Copyright © 2024 AppLovin Corporation. All rights reserved.
+//
+
+// Empty file for SPM wrapper target
```

Comment:
For my understanding, is this just to keep the `AppLovinSDKResources` target with all the linker settings?

Reply:
Yep exactly